pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -72,22 +94,54 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Replica Quiescence" sources={storeSources}>
+    <LineGraph
+      title="Replica Quiescence"
+      sources={storeSources}
+      tooltip={(
+        <div>
+        Number of replicas and quiesced replicas {tooltipSelection}.
+        <br/>
+        Quiesced replicas have no pending reads or writes and have temporarily disabled consensus protocols.
+        </div>
+      )}
+    >
       <Axis label="replicas">
         <Metric name="cr.store.replicas" title="Replicas" />
         <Metric name="cr.store.replicas.quiescent" title="Quiescent" />
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Range Operations" sources={storeSources}>
+    <LineGraph title="Range Operations" sources={storeSources}
+      tooltip = {`Ranges with split, add, or remove operations ${tooltipSelection}.`}
+    >
       <Axis label="ranges">
         <Metric name="cr.store.range.splits" title="Splits" nonNegativeRate />
         <Metric name="cr.store.range.adds" title="Adds" nonNegativeRate />
         <Metric name="cr.store.range.removes" title="Removes" nonNegativeRate />
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Snapshots" sources={storeSources}>
+    <LineGraph
+      title="Snapshots"
+      sources={storeSources}
+      tooltip={(
+        <div>
+          Snapshots {tooltipSelection} <br/>
+          When a node is far behind the log file for a range, the cluster can
+          send it a snapshot of the range and it can start following the log from there.
```

Comment:
from @Amruta-Ranade : this needs to be simplified to make it easier for users outside of CRL to understand.